src/patternfly/components/AppLauncher/app-launcher-separator.hbs

```diff
@@ -1,7 +1,10 @@
 <{{#if app-launcher-separator--IsHr}}hr{{else}}li{{/if}} class="pf-c-app-launcher__separator{{#if app-launcher-separator--modifier}} {{app-launcher-separator--modifier}}{{/if}}"
   {{#if app-launcher-separator--attribute}}
 	  {{{app-launcher-separator--attribute}}}
-  {{/if}}>
+  {{/if}}
+  {{#unless app-launcher-separator--IsHr}}
```

Comment:
Were you thinking of permanently keeping an app-launcher-separator element? or replacing it with the divider when the divider isn't experimental

Reply:
We'll replace it. Dana mentioned that we shouldn't use or ship experimental features in non-experimental components. I have another issue to update the nav dividers to separators. We can update these components to use the divider once it's moved out of experimental.